src/Comments/Comments.js

```diff
@@ -16,43 +16,55 @@ export class Comments extends Component  {
     updateValue = (event) => {
       event.preventDefault();
       this.setState({[event.target.name]: event.target.value});
-      console.log(event.target)
     }
 
-    postComment = () => {
+    clearForm = () => {
+      this.setState({comment: ''})
+    }
+      
+    postComment = (event) => {
+      event.preventDefault();
         if(!this.state.comment || !this.props.name){
             return false
         }
+        this.clearForm()
         this.props.newComment(this.props.movieId, this.state.comment)
-    }
+        this.props.retrieveComments(this.props.movieId)
+  }
+
+  
 
   render() {
     const { movieComments } = this.props
-    console.log(movieComments)
       return (
         <div className="comment-container">
           
           <div className="inner-comment">
             <h1 className="comment-title">Comments</h1>
-            <input
-              className="input-text"
-              type="text"
-              name="comment"
-              role="comment-section"
-              type="comment-section"
-              maxLength = "120"
-              placeholder="Add your comment here"
-              onChange={this.updateValue}
-              value={this.state.commentValue}
-            />
+            <form className="form-comment">
+              <input
+                className="input-text"
+                type="text"
+                name="comment"
+                role="comment-section"
+                type="comment-section"
+                maxLength = "120"
+                placeholder="Add your comment here"
+                onChange={this.updateValue}
+                value={this.state.commentValue}
+              />
+
+              <button 
+              role="commnt-button"
+              type="commnt-button"
+              onSubmit={this.postComment}
+              onClick={this.postComment}
```

Comment:
Nice! This just reminded me, did we ever make the login page utilize onSubmit as well?